packages/eth-deposit/package.json

```diff
@@ -3,16 +3,21 @@
   "license": "Apache-2.0",
   "version": "1.0.0",
   "scripts": {
-    "build": "hardhat compile",
+    "build": "hardhat compile && tsc",
+    "build-esm": "hardhat compile && tsc -p tsconfig-esm.json",
     "depositETH": "hardhat run scripts/exec.js"
```

Comment:
This needs to be changed to `hardhat run scripts/exec.ts`

Reply:
Nice catch ! I just changed it.